pkg/cmd/pr/shared/templates.go

```diff
@@ -35,7 +41,19 @@ func (t *issueTemplate) Body() []byte {
 	return []byte(t.Gbody)
 }
 
-func listIssueTemplates(httpClient *http.Client, repo ghrepo.Interface) ([]issueTemplate, error) {
+func (t *pullRequestTemplate) Name() string {
+	return t.Gname
+}
+
+func (t *pullRequestTemplate) NameForSubmit() string {
+	return t.Gname
```

Comment:
Do PR templates have an identifier used for submitting the PR like issues do? I would say not, because the `createPullRequest` mutation doesn't seem to accept one. In that case, this method should return a blank string to indicate that there is nothing to submit.